prov/efa/src/efa_hmem.c

```diff
@@ -138,7 +149,8 @@ static int efa_hmem_support_status_update_neuron(struct efa_hmem_support_status
 	ibv_mr = ibv_reg_mr(g_device_list[0].ibv_pd, ptr, len, ibv_access);
 	if (!ibv_mr) {
 		EFA_WARN(FI_LOG_DOMAIN,
-			 "Failed to register Neuron buffer with the EFA device, FI_HMEM transfers that require peer to peer support will fail.\n");
+		         "Failed to register Neuron buffer with the EFA device,"
+		         "FI_HMEM transfers that require peer to peer support will fail.\n");
```

Comment:
Probably want to add a space following the comma here

Reply:
done